server/app/views/questiontypes/PhoneQuestionRenderer.java

```diff
@@ -0,0 +1,86 @@
+import views.components.FieldWithLabel;
+import views.components.SelectWithLabel;
+import views.style.ReferenceClasses;
+
+public class PhoneQuestionRenderer extends ApplicantSingleQuestionRenderer {
+  PhoneQuestionRenderer(ApplicantQuestion question) {
+    super(question);
+  }
+
+  @Override
+  public String getReferenceClass() {
+    return "cf-question-phone";
+  }
+
+  @Override
+  protected DivTag renderInputTag(
+      ApplicantQuestionRendererParams params,
+      ImmutableMap<Path, ImmutableSet<ValidationErrorMessage>> validationErrors,
+      ImmutableList<String> ariaDescribedByIds) {
+    PhoneQuestion phoneQuestion = question.createPhoneQuestion();
+
+    Messages messages = params.messages();
+
+    SelectWithLabel countryCodeField =
+        (SelectWithLabel)
+            new SelectWithLabel()
+                .addStyleClass("py-15")
+                .setFieldName(phoneQuestion.getCountryCodePath().toString())
+                .setValue(phoneQuestion.getCountryCodeValue().orElse(""))
+                .setLabelText(messages.at(MessageKey.PHONE_LABEL_COUNTRY_CODE.getKeyName()))
+                .setOptionGroups(
+                    ImmutableList.of(
+                        SelectWithLabel.OptionGroup.builder()
+                            .setLabel(messages.at(MessageKey.PHONE_LABEL_COUNTRY_CODE.getKeyName()))
+                            .setOptions(countryOptions())
+                            .build()))
+                .setFieldErrors(
+                    messages,
+                    validationErrors.getOrDefault(
+                        phoneQuestion.getCountryCodePath(), ImmutableSet.of()))
+                .setId(ReferenceClasses.PHONE_COUNTRY_CODE);
+
+    FieldWithLabel phoneField =
+        FieldWithLabel.input()
+            .setPlaceholderText("(xxx) xxx-xxxx")
+            .setFieldName(phoneQuestion.getPhoneNumberPath().toString())
+            .setValue(phoneQuestion.getPhoneNumberValue().orElse(""))
+            .setLabelText(messages.at(MessageKey.PHONE_LABEL_PHONE_NUMBER.getKeyName()))
+            .setFieldErrors(
+                messages,
+                validationErrors.getOrDefault(
+                    phoneQuestion.getPhoneNumberPath(), ImmutableSet.of()))
+            .setAriaDescribedByIds(ariaDescribedByIds)
+            .setScreenReaderText(question.getQuestionTextForScreenReader())
+            .setId(ReferenceClasses.PHONE_NUMBER);
+
+    if (!validationErrors.isEmpty()) {
+      countryCodeField.forceAriaInvalid();
+      phoneField.forceAriaInvalid();
+    }
+
+    return div()
+        .withClasses("grid")
+        .with(countryCodeField.getSelectTag(), phoneField.getInputTag());
+  }
+
+  private static ImmutableList<SelectWithLabel.OptionValue> countryOptions() {
+    return ImmutableList.of(
+        SelectWithLabel.OptionValue.builder().setLabel("United States").setValue("US").build(),
+        SelectWithLabel.OptionValue.builder().setLabel("Canada").setValue("CA").build());
+  }
```

Comment:
**suggestion (blocking):** any reason for this to be a static method and not a constant?
```suggestion
  private static final ImmutableList<SelectWithLabel.OptionValue> COUNTRY_OPTIONS = ImmutableList.of(
        SelectWithLabel.OptionValue.builder().setLabel("United States").setValue("US").build(),
        SelectWithLabel.OptionValue.builder().setLabel("Canada").setValue("CA").build());
```